appium/README.md

```diff
@@ -24,10 +25,18 @@ To run a particular test:
 
 To run all tests: `npm run-script test.live.all` or `npm run-script test.mock.all`
 
+## Write and debug tests
+Tips for debugging:
+- For debugging purposes instead of running `bundle exec fastlane build` you can copy `FlowCrypt.app` from `/DerivedData/FlowCrypt-.../Build/Products/Debug-iphonesimulator` (In Xcode open `Products` folder -> `FlowCrypt` -> `Show in Finder`) to appium folder. 
+- Remove contents of `appium/tmp` before test execution. 
+- Execute tests and check `appium/tmp` for troubleshooting.
+- You can change log level to debug/error inside `appium/config/wdio.shared.conf.js`. `logLevel: 'debug'`.
+- You can inspect accessibility identifiers of ui elements with `appium-inspector`.
```

Comment:
can add "if appium doesn't even start simulator where it used to work, try deleting `node_modules` folder and running `npm install`. Also check your nodejs version is 16 with `node --version`"